romanesco/plugins/swift/executor.py

```diff
@@ -0,0 +1,112 @@
+import os
+import re
+import select
+import subprocess
+import sys
+import tempfile
+
```

Comment:
It would be good to pull out the stuff inside `_run_swift_process` since it's the exact same as the docker one, and is generally useful for anything running the task as a subprocess. We could put that in the `utils` module in core perhaps.